src/AnchoredOverlay/AnchoredOverlay.tsx

```diff
@@ -10,7 +10,12 @@ export interface AnchoredOverlayProps extends Pick<OverlayProps, 'height' | 'wid
    * A custom function component used to render the anchor element.
    * Will receive the selected text as `children` prop when an item is activated.
    */
-  renderAnchor: <T extends React.HTMLAttributes<HTMLElement>>(props: T) => JSX.Element
+  renderAnchor: <T extends React.HTMLAttributes<HTMLElement>>(props: T) => JSX.Element | null
```

Comment:
Should we allow a direct value of `null` instead of `() => null`?  That way it isn't seen as a prop change every time the consumer re-renders.  If so, we may want to get a little fancier with the types and _require_ an `anchorRef` when `renderAnchor` is `null`